README.md

```diff
@@ -763,7 +763,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Exponentiation">Exponentiation (Recursive)</a></td>
             <td> <!-- C -->
-                <a href="./src/c/ExponenciacaoRecursiva.c">
+                <a href="./src/c/RecursiveExponentiation.c">
```

Comment:
Maybe we could do that here too, like `ExponentiationRecursive.c`

Reply:
If the idea is for similar algorithms to be close, using recursive as a prefix seems like a better idea to me

Reply:
this also seems questionable about the recursion algorithms being together